slack_status_cli/main.py

```diff
@@ -237,7 +247,7 @@ def run():
         )
         logger.info("✨ Status set to '%s' %s", new_status, new_expiry)
     except Exception as e:
-        logger.exception("Could not set status: %s", str(e))
+        logger.error("🔥 Could not set status: %s", str(e))
```

Comment:
Changed from `exception` to `error` -- we do not need to expose the trace to this logger.